mempool/cat/reactor.go

```diff
@@ -587,6 +618,64 @@ func (memR *Reactor) tryRequestQueuedTx(entry *pendingSeenTx) bool {
 	return memR.requests.ForTx(entry.txKey) != 0
 }
 
+func (memR *Reactor) heightSignalLoop() {
```

Comment:
this feels like more of a hack than a solution, however it is a relatively bullet proof way to ensure that nodes are always checking on txs they need to be querying

personally I'm still not sold that we need this

Reply:
Yeah agree that that would be cleaner